Slavahaim/hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def get_middle(str)
+  str[(str.size - 1) / 2..str.size / 2]
```

Comment:
We can extract `str.size - 1` and `2..str.size / 2` into varibles and make a meaning of the varibles smth like `last_element` and `middle_element`. 
Try to avoid `magic number` like in `2..str.size / 2`. It's corrected by extracting into variables with meaning name. It makes a code more readable